control-plane/subcommand/connect-init/command.go

```diff
@@ -179,6 +179,12 @@ func (c *Command) Run(args []string) int {
 				c.logger.Info("Check to ensure a Kubernetes service has been created for this application." +
 					" If your pod is not starting also check the connect-inject deployment logs.")
 			}
+			if len(serviceList) > 2 {
+				c.logger.Error("There are multiple Consul services registered for this pod when there should only be one." +
+					" Check if there are multiple Kubernetes services pointing at this pod and add the label" +
+					" `consul.hashicorp.com/service-ignore: \"true\"` to services which should not be used by Consul for handling requests.")
```

Comment:
I think it's good to be very explicit here so as not to leave room for users to shoot themselves in the feet.

```suggestion
				c.logger.Error("There are multiple Consul services registered for this pod when there must only be one." +
					" Check if there are multiple Kubernetes services selecting this pod and add the label" +
					" `consul.hashicorp.com/service-ignore: \"true\"` to all services except the one used by Consul for handling requests.")
```